python/ray/data/tests/test_dataset.py

```diff
@@ -1235,20 +1235,24 @@ def test_to_pandas_refs(ray_start_regular_shared):
 
 
 def test_to_numpy(ray_start_regular_shared):
+    # Simple Dataset
+    ds = ray.data.range(10)
+    arr = np.concatenate(ray.get(ds.to_numpy_refs()))
+    np.testing.assert_equal(arr, np.arange(0, 10))
+
     # Tensor Dataset
     ds = ray.data.range_tensor(10, parallelism=2)
     arr = np.concatenate(ray.get(ds.to_numpy_refs(column="value")))
-    np.testing.assert_equal(arr, np.expand_dims(np.arange(0, 10), 1))
+    np.testing.assert_equal(arr, np.arange(0, 10))
```

Comment:
Not sure why this originally returned the tensor column wrapped in an extra unit dimension? This is a change in semantics, but the previous semantics seems like a bug to me. Given that we're not aware of any users using tensor datasets, I'd like to squeak in this fix.

Reply:
I think that's because the original tensor had shape (1,), not (). So this seems correct.

What code was changed to cause this?

Reply:
90% sure the change was `return self._table[column].chunk(0).to_numpy()` --> `return self._table[column].to_numpy()`, so just doing an ndarray conversion on the `ChunkedArray` instead of on a single chunk `Array`.

Reply:
>I think that's because the original tensor had shape (1,), not (). So this seems correct.

I'd think that the column selection with `ds.to_numpy_refs(column="value")` would take the `(1, num_rows)` single-column table and return a `(num_rows,)` 1D ndarray.